<p>Sharing some naming conventions is a key point to make it possible for a team to efficiently collaborate. This rule allows to check that field
names match a provided regular expression.</p>
<h2>Noncompliant Code Example</h2>

<p>共享一些命名约定是使团队有效协作的关键点。 此规则允许检查字段名称是否与提供的正则表达式匹配。</ p>
<h2>不符合规范的代码示例</ h2>
<p>使用默认正则表达式 <code>^[_a-z][_a-z0-9]*$</code>:</p>
<pre>
class MyClass:
  myField = 1
</pre>
<h2>合规的解决方案</h2>
<pre>
class MyClass:
  my_field = 1
</pre>

