<p>通过对比，每个Python函数都返回一些东西，即使它是<code> None </code>值，可以通过省略<code> return </code>语句或显式地隐式返回。</p>

<p>The <code>__init__</code> method is required to return <code>None</code>. A <code>TypeError</code> will be raised if the <code>__init__</code>
method either <code>yield</code>s or <code>return</code>s any expression other than <code>None</code>. Returning some expression that evaluates to
<code>None</code> will not raise an error, but is considered bad practice.</p>
<h2>Noncompliant Code Example</h2>

（<p> <code> __ init __ </ code>方法需要返回<code> None </ code>。 如果<code> __ init __ </ code>方法<code>产生</ code> s或<code>返回</ code> s代码以外的任何表达式>无会引发<code> TypeError </code></代码>。 返回一些计算结果为<code> None </ code>的表达式不会引发错误，但被认为是不好的做法。</p> 
<h2>不符合规范的代码示例</ h2>）
<pre>
class MyClass(object):
    def __init__(self):
        self.message = 'Hello'
        return self  # Noncompliant
</pre>
<h2>合规的解决方案</h2>
<pre>
class MyClass(object):
    def __init__(self):
        self.message = 'Hello'
</pre>

